region/crateria/central.json

```diff
@@ -744,6 +768,18 @@
           "locks": null
         }
       ],
+      "enemies": [
+        {
+          "name": "Grey Space Pirate (wall)",
+          "quantity": 11,
+          "homeNodes": [ 5 ]
+        },
+        {
+          "name": "Space Pirate (fighting)",
```

Comment:
Isn't that the name of the Metal Pirates in Lower Norfair?

Reply:
Yeah I knew messed up with that somewhere